pkg/clusterregistry/testing/testserver.go

```diff
@@ -49,7 +50,8 @@ func StartTestServerOrDie(t *testing.T) (*restclient.Config, TearDownFunc) {
 		if err == nil {
 			return config, td
 		}
-		if err != nil && !strings.Contains(err.Error(), "bind") {
+		if !strings.Contains(err.Error(), "bind") {
+			t.Fatalf("Failed to launch server: %v", err)
 			break
```

Comment:
Not entirely sure if `t.Fatalf` replaced `break`. I still see `break` here. But you can fix this in a follow up PR.

Reply:
Whoops, that was not intentional! It's easy enough to fix up here, so I'll do that.

Reply:
Thanks for fixing it.